lisa/features/security_profile.py

```diff
@@ -114,6 +114,9 @@ def can_disable(cls) -> bool:
     def enabled(self) -> bool:
         return True
 
+    def get_settings(self) -> schema.FeatureSettings:
+        return self._settings
```

Comment:
It can be moved to base feature class, but use a deep copy to make it read only. Also add comments it's a read only value, the changes won't be effctive.

Reply:
Moved